### PR TITLE
Updates to Quick-Start Guide V2 now using scripts repo

### LINKED ISSUE #199: C4B QuickStart Guide V2
The Chocolatey for Business (C4B) QuickStart Guide is beginning to show its age, for various reasons:

- Most users just want to stand up a single server for POC (similar to QDE); current guide does not account for this
- No coverage of CCM setup
- Changes in licensed package availability means no more need to download packages from emails
- The `chocolatey-c4b-quickstart` and `chocolatey-nexus-setup` packages need to be updated (usually by @gep13) at every new release; which is inefficient
- Would be better to ease the setup process with some user interactivity (forms and file dialogs), along with some more validation (is Chocolatey installed, validation of license. etc.)
- Scripts would give us a little more freedom to accomplish the above and simplify for the user, so that they could directly copy and paste code and not necessarily have to adjust values if they accept the defaults 
- Some of the learnings form QDE could be incorporated into this process

Work is already underway for these changes by myself and @ryanrichter94, with input from @steviecoaster. I am logging the issue here, and will submit a WIP PR shortly.

### PR DESCRIPTION
**Pull request by adilio**
Closes #199 .

This is the biggest update to the C4B QuickStart Guide (QSG) in about 2 years!

This update includes:
- Automated script setup, allowing users to simply copy-and-paste code
- Nexus repository setup
- **New** BYO Server; we have tested and support all current non-EOL Windows Server  versions (2016, 2019, 2022) on your infrastructure-of-choice
- **New** Complete CCM setup, including MS SQL Express
- **New** License validation and license package creation
- **New** Jenkins setup, along with job templates
- **New** SSL Setup, which **now accepts custom certs**

It's also worth mentioning that this guide bases most of its automation on code contained in the `choco-quickstart-scripts` repo here:
https://github.com/chocolatey/choco-quickstart-scripts

### CODE REVIEW

**Review by Windos — CHANGES_REQUESTED** [review]
Just a very quick grammar check and ensure the preview works. A few minor things to tweak from that.

**Review by gep13 — CHANGES_REQUESTED** [review]
Just a few suggestions/questions that occurred to me as I was reading through this.  On the whole, looks really good to me though, well done!

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md [hunk 1]
```diff
@@ -6,233 +6,229 @@ Description: Get up and running quickly with Chocolatey for Business
...
+    > <details>
+    > <summary><strong>What does this script do? (click to expand)</strong></summary>
+    > <ul class="list-style-type-disc">
+    > <li>Installs Chocolatey client (chocolatey), using a script from your raw "choco-install" repository</li>
+    > <li>Runs the "ClientSetup.ps1" script from your raw "choco-install" repository, which does the following:</li>
+    > <li>Licenses Chocolatey by installing the license package (chocolatey-license) created during QDE setup</li>
+    > <li>Installs the Chocolatey Licensed Extension (chocolatey.extension) without context menus</li>
+    > <li>Configures "ChocolateyInternal" source</li>
+    > <li>Disables access to the "chocolatey" public Chocolatey Community Repository (CCR)</li>
+    > <li>Configures Self-Service mode and installs Chocolatey GUI (chocolateygui)</li>
```
[the remark this answers is not in the document]
Comment (adilio) [reply input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md:214]: We are not, but we do download it to the repo; should we?

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md [hunk 2]
```diff
@@ -6,233 +6,229 @@ Description: Get up and running quickly with Chocolatey for Business
...
+    .\Register-C4bEndpoint.ps1
     ```
 
-1. Install the `chocolatey.extension` package:
+    > <details>
+    > <summary><strong>What does this script do? (click to expand)</strong></summary>
+    > <ul class="list-style-type-disc">
+    > <li>Installs Chocolatey client (chocolatey), using a script from your raw "choco-install" repository</li>
+    > <li>Runs the "ClientSetup.ps1" script from your raw "choco-install" repository, which does the following:</li>
+    > <li>Licenses Chocolatey by installing the license package (chocolatey-license) created during QDE setup</li>
```
[the remark this answers is not in the document]
Comment (adilio) [reply input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md:210]: Good point

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md [hunk 3]
```diff
@@ -6,233 +6,229 @@ Description: Get up and running quickly with Chocolatey for Business
...
+    > <details>
+    > <summary><strong>What does this script do? (click to expand)</strong></summary>
     > <ul class="list-style-type-disc">
-    >   <li>Remove unwanted demo repositories</li>
-    >   <li>Add a "choco-hosted" Nuget repository</li>
-    >   <li>Add a "choco-install" raw repository</li>
-    >   <li>Add an "Install.ps1" script to the "choco-install" raw repository</li>
-    >   <li>Enable the Nuget API key realm</li>
-    >   <li>Output the API key for the Nexus server (copy this key for a later step)</li>
+    > <li>Installs MS SQL Express</li>
```
[the remark this answers is not in the document]
Comment (adilio) [reply input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md:128]: Good point, will do.

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adilio) [reply input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md]: My bad, we already were. 😂 
https://github.com/chocolatey/choco-quickstart-scripts/blob/4ebb77e67440297ca4e4150d5aed1f68229d6adb/scripts/ClientSetup.ps1#L108
Added that to the doc in this last PR.

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md [hunk 4]
```diff
@@ -6,233 +6,233 @@ Description: Get up and running quickly with Chocolatey for Business
...
-1. The administrative password for the Nexus web UI can be found in the following file:
+    > :warning:**WARNING**: **Only if** you choose to run this on a **Windows Server 2016** VM, you will **require** a **reboot** before IIS is completely installed. The script above will notify you of this. Once the reboot is complete and you log back in, you will also have to paste and run the following code in a PowerShell Administrator console:
 
-    ```cmd
-    C:\ProgramData\sonatype-work\nexus3\admin.password
-    ```
+    > ```powershell
+    > Set-Location "$env:SystemDrive\choco-setup\files\scripts"
+    > .\Start-C4bCcmSetup2.ps1
+    > ```
```
Comment (JPRuskin) [inline input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md:142]: When in quoteblock _and_ codeblock, this renders like this:
image

I would recommend just a codeblock?
```suggestion
    ```powershell
    Set-Location "$env:SystemDrive\choco-setup\files\scripts"
    .\Start-C4bCcmSetup2.ps1
    ```
```

Comment (JPRuskin) [reply input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md:142]: Ah, but I assume we want to bundle it in with the warning above. Hm. In that case, removing the blank line between the two quotes will result in this: 
image

(suggestion above)

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md [hunk 5]
```diff
@@ -6,233 +6,233 @@ Description: Get up and running quickly with Chocolatey for Business
...
 
-1. You will require Google Chrome in order to login to the Nexus web UI (site can appear unresponsive in IE). From an Administrator PowerShell console, run:
+    <br>
 
-    ```powershell
-    choco install googlechrome -y
-    ```
-
-1. The administrative password for the Nexus web UI can be found in the following file:
+    > :warning:**WARNING**: **Only if** you choose to run this on a **Windows Server 2016** VM, you will **require** a **reboot** before IIS is completely installed. The script above will notify you of this. Once the reboot is complete and you log back in, you will also have to paste and run the following code in a PowerShell Administrator console:
```
Comment (JPRuskin) [inline input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md:138]: The warning and code block are separated and render awkwardly. This should solve that!

```suggestion
```

**Review by JPRuskin — CHANGES_REQUESTED** [review]
Hopefully some of this is helpful and not just me nitpicking - would suggest changing the blockquote stuff (if I guessed your intent right), at least, as it looks quite odd on my preview (though if Gary didn't notice it, I worry that it's not an issue anywhere else 😅).

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md [hunk 6]
```diff
@@ -6,233 +6,233 @@ Description: Get up and running quickly with Chocolatey for Business
...
-    choco install chocolatey-c4b-quickstart --source . -y --params "'/LicenseFile:<FULL_PATH_TO_LICENSE_FILE>'"
+    Set-ExecutionPolicy Bypass -Scope Process -Force
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::tls12
+    $QuickStart = 'https://raw.githubusercontent.com/chocolatey/choco-quickstart-scripts/main/Start-C4bSetup.ps1'
+    Invoke-Expression -Command ((New-Object System.Net.WebClient).DownloadString($QuickStart))
     ```
 
-    > :memo: **NOTE** We have built this package to SIMPLIFY the initial setup greatly; it will perform the following:
+    > <details>
+    > <summary><strong>What does this script do? (click to expand)</strong></summary>
```
[the remark this answers is not in the document]
Comment (JPRuskin) [reply input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md:81]: I've added what I think to be a suggested fix for this in various places, which ends up looking like this:

image

and expanding to this:
image

**Comment by adilio** [comment]
Heya @JPRuskin, thanks for all the GREAT input and call. 🙏🏼 

I have incorporated all the changes we've discussed in this last commit.

Ready for review.

File: input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (adilio) [reply input/en-us/guides/organizations/chocolatey-for-business-quick-start-guide.md:215]: This breaks the formatting, btw. Had to revert back.

**Review by JPRuskin — DISMISSED** [review]
Looks great! Sorry for my confusion around the quotes.

**Comment by adilio** [comment]
Hey @JPRuskin, now it's my turn to apologize! For some reason, I accidentally dismissed your review. Could you re-review and approve if acceptable?
Many thanks! 🙏🏼

**Review by JPRuskin — APPROVED** [review]
LGTM!

**Review by gep13 — DISMISSED** [review]
LGTM!

Only thing I would suggest is that the commits can/should be squashed down into a single commit, as they are really all changes that are based on the feedback that was received.

**Review by ryanrichter94 — DISMISSED** [review]
LGTM

**Review by steviecoaster — APPROVED** [review]
LGTM 🚀

**Pull request merged**